autoPyTorch/api/base_task.py

```diff
@@ -492,7 +496,7 @@ def _do_dummy_prediction(self):
             logger=self._logger,
             cost_for_crash=get_cost_of_crash(self._metric),
             abort_on_first_run_crash=False,
-            initial_num_run=self._num_run,
+            initial_num_run=self._num_run if self._num_run is not None else 1,
```

Comment:
Ho can self._num_run be not 1? Do you think it is better to make num_run an argument to dummy prediction?

Reply:
I made it this way as in traditional classification the num_run depends on the number of available classifiers, which could be dynamic, therefore, this attribute of the class gets updated after it is run. and this if statement is to comply with mypy